controller-server/src/main/java/com/yahoo/vespa/hosted/controller/application/EndpointList.java

```diff
@@ -67,16 +67,14 @@ public static EndpointList of(Stream<Endpoint> endpoints) {
     }
 
     /** Returns the default global endpoints in given system. Default endpoints are served by a pre-provisioned routing layer */
-    public static EndpointList defaultGlobal(ApplicationId application, SystemName system) {
-        // Rotation name is always default in the routing layer
-        RotationName rotation = RotationName.from("default");
+    public static EndpointList defaultGlobal(ApplicationId application, EndpointId endpointId, SystemName system) {
```

Comment:
This should be renamed since "default" is no longer implied. Just `create` perhaps?